.github/workflows/sourcify.yml

```diff
@@ -0,0 +1,163 @@
+name: "sourcify"
```

Comment:
WDYT of naming them `sourcify_all_chains` and `sourcify_single_chain` to distinguish?